### PR TITLE
Added mutual SSL authentication handling for android

### PR DESCRIPTION
**Pull request by janakamarasena**

### CODE REVIEW

**Review by chaturadilan — APPROVED** [review]

File: components/webapp-authenticator-framework/org.wso2.carbon.webapp.authenticator.framework/src/main/java/org/wso2/carbon/webapp/authenticator/framework/authenticator/CertificateAuthenticator.java [hunk 1]
```diff
@@ -164,4 +164,4 @@ public String getProperty(String name) {
         return null;
     }
 
-}
+}
```
Comment (Kanapriya) [inline components/webapp-authenticator-framework/org.wso2.carbon.webapp.authenticator.framework/src/main/java/org/wso2/carbon/webapp/authenticator/framework/authenticator/CertificateAuthenticator.java:167]: Add new line

File: components/apimgt-extensions/org.wso2.carbon.apimgt.handlers/src/main/java/org/wso2/carbon/apimgt/handlers/AuthenticationHandler.java [hunk 2]
```diff
@@ -205,4 +231,4 @@ private String getDeviceType(String url) {
         map.put(CONTENT_TYPE, "application/json");
         return map;
     }
-}
+}
```
Comment (Kanapriya) [inline components/apimgt-extensions/org.wso2.carbon.apimgt.handlers/src/main/java/org/wso2/carbon/apimgt/handlers/AuthenticationHandler.java:234]: Add new line here

Comment (janakamarasena) [reply components/apimgt-extensions/org.wso2.carbon.apimgt.handlers/src/main/java/org/wso2/carbon/apimgt/handlers/AuthenticationHandler.java:234]: Fixed with https://github.com/wso2/carbon-device-mgt/pull/1278

File: components/webapp-authenticator-framework/org.wso2.carbon.webapp.authenticator.framework/src/main/java/org/wso2/carbon/webapp/authenticator/framework/authenticator/CertificateAuthenticator.java (hunk 1 above)
Comment (janakamarasena) [reply components/webapp-authenticator-framework/org.wso2.carbon.webapp.authenticator.framework/src/main/java/org/wso2/carbon/webapp/authenticator/framework/authenticator/CertificateAuthenticator.java:167]: Fixed with https://github.com/wso2/carbon-device-mgt/pull/1278

**Pull request merged**